magefile.go

```diff
@@ -153,6 +153,7 @@ func Build() error {
 	err := sh.RunWith(
 		getBuildEnvironment(), goExecutableName, "build",
 		"-o", ttExecutableName,
+		"-tags=dev go_tarantool_ssl_disable",
```

Comment:
What does `dev` build tag do?

Reply:
Disables linking with OpenSSL. We do not need this in TT. Also this needed for static build.

Reply:
Fixed